cards/card_component.js

```diff
@@ -16,6 +16,10 @@ BaseCard["{{componentName}}"] = class extends ANSWERS.Component {
    * only if such a toggle is present in the DOM.
    */
   onMount() {
+    //polyfill for image styling (object-fit) in ie11
```

Comment:
I was thinking it might be worth it to run it on the entire card instead of just this css class, on the off chance somebody renames their card css classes, but this is probably fine for now!